website/blog/2023-10-11-silver.md

```diff
@@ -0,0 +1,474 @@
+---
+author: Katarzyna Marek
+title: Metals v1.1.0 - Silver
+authorImageURL: https://github.com/kasiaMarek.png
+---
+
+We're happy to announce the release of Metals v1.1.0, which brings stability fixes and takes another step towards bettering error detection and error reporting.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">101</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">92</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/60?closed=1]
+(https://github.com/scalameta/metals/milestone/60?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
```

Comment:
I think once https://github.com/scalameta/metals/pull/5747 is merged you can actually remove the 

> with the help from Lunatech

part since we're not actively working on this at the moment.